src/components/Table/components/ActionsCell/ActionsCell.tsx

```diff
@@ -27,10 +29,11 @@ function ActionsCell({ actions, row, disablePortal = false }: ActionsCellProps)
     .map(({ action, content, disabled, ...rest }) => ({
       ...rest,
       content: content instanceof Function ? content(row.original) : content,
-      disabled: disabled instanceof Function ? disabled(row.original) : disabled,
+      disabled:
+        disabled instanceof Function ? disabled(row.original) : disabled,
       ...(action && {
         onClick: () => {
-          action({ ...row.original, ...row.state })
+          action(row.original)
```

Comment:
this one because there's no row.state anymore? same as with cell.state in ApiCallCell?

Reply:
yes